scripts/apps/search/components/fields/state.tsx

```diff
@@ -49,7 +50,7 @@ export const state: React.StatelessComponent<Pick<IPropsItemListInfo, 'item'>> =
                 className={'state-label state-' + props.item.state}
                 key="state"
             >
-                {title}
+                {text}
```

Comment:
What is this about?

Reply:
the title and the text are [the same](https://github.com/superdesk/superdesk-client-core/pull/3563/files/ff69ce08e788ad408b0daac980cea67eb884aa14..0b72839c16faa220b68a40233925e3ee8c0a36fc#diff-c40a8b98bc23bffcf218fff661622d58R37) unless the item is scheduled, then the title [contains](https://github.com/superdesk/superdesk-client-core/pull/3563/files/ff69ce08e788ad408b0daac980cea67eb884aa14..0b72839c16faa220b68a40233925e3ee8c0a36fc#diff-c40a8b98bc23bffcf218fff661622d58R42-L43) the full time of the scheduled date, which is not meant to appear on the label text